javascript-operators-and-expressions/main.js

```diff
@@ -38,5 +38,5 @@ console.log('value of isSparta', isSparta);
 console.log('type of isSparta', typeof isSparta);
 
 var motto = fullName;
-console.log(motto, ' is the GOAT');
+console.log('value of motto', motto, 'is the GOAT');
```

Comment:
This is still not correct, you are not using the addition assignment operator at all